src/function/table/system/duckdb_extensions.cpp

```diff
@@ -190,25 +187,24 @@ void DuckDBExtensionsFunction(ClientContext &context, TableFunctionInput &data_p
 	while (data.offset < data.entries.size() && count < STANDARD_VECTOR_SIZE) {
 		auto &entry = data.entries[data.offset];
 
+		int col = 0;
```

Comment:
```suggestion
		idx_t col = 0;
```
Otherwise warning turned error on OSX Debug